### PR TITLE
Brush up some frontend layouts

### PR DESCRIPTION
**Pull request by Jonathanlau92**
# Pull Request Description 

Task(s) you did:
- Add icons to friend requests
- Placeholder for forms

How did you test it? : 
- By naked eye

Remarks:
- Nil

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -49,26 +48,29 @@
...
+              <div class="card-body pt-1">
+                <span>Meet a new friend today!</span>
               </div>
             <% end %>
-          <% else %>
-            <div class="card-body pt-1">
-              <span>Meet a new friend today!</span>
-            </div>
-          <% end %>
+          </div>
```
Comment (Jonathanlau92) [inline app/views/users/show.html.erb:93]: @TimothyHou testing on slack! please review!

**Comment by TimothyHou** [comment]
hey! 2 things:
1) For those with friendship status requested, i made the name into a link. So people can view the profile of people who request to be their friend.

2) Is it possible to further separate those with friendship status pending and requested? It was quite confusing and take some time to get used to.
Capture

Other than that it looks good, thanks!

**Comment by Jonathanlau92** [comment]
@TimothyHou Further separate means what? The ideal goal is still to use notification bell, so perhaps have to find a temporary solution to distinguish both pending and accepted request (im assuming thats what you meant? haha)

**Comment by TimothyHou** [comment]
yupyup! cause now Pending friend request got 2 types right? One is 'people want to friend you', the other one is 'you want to friend people'. Maybe can just do simple split into 2 cards, 'Received friend requests' and 'Sent friend requests', like linkedin liddat: 
Capture. 
Then next time can change it to notification mode.

**Comment by Jonathanlau92** [comment]
@TimothyHou changed done! Please review thanks!

**Comment by TimothyHou** [comment]
yup it's ok! thanks. i'll help u merge

**Pull request merged**